builtin/types/chainconfig/chainconfig.pb.go

```diff
@@ -825,8 +825,47 @@ func (m *AppStoreConfig) GetNumEvmKeysToPrune() uint64 {
 	return 0
 }
 
+type EvmConfig struct {
+	GasLimit             uint64   `protobuf:"varint,1,opt,name=gas_limit,json=gasLimit,proto3" json:"gas_limit,omitempty"`
+	XXX_NoUnkeyedLiteral struct{} `json:"-"`
+	XXX_unrecognized     []byte   `json:"-"`
+	XXX_sizecache        int32    `json:"-"`
+}
+
+func (m *EvmConfig) Reset()         { *m = EvmConfig{} }
+func (m *EvmConfig) String() string { return proto.CompactTextString(m) }
+func (*EvmConfig) ProtoMessage()    {}
+func (*EvmConfig) Descriptor() ([]byte, []int) {
+	return fileDescriptor_chainconfig_361fd833d207eaa5, []int{18}
+}
+func (m *EvmConfig) XXX_Unmarshal(b []byte) error {
+	return xxx_messageInfo_EvmConfig.Unmarshal(m, b)
+}
+func (m *EvmConfig) XXX_Marshal(b []byte, deterministic bool) ([]byte, error) {
+	return xxx_messageInfo_EvmConfig.Marshal(b, m, deterministic)
+}
+func (dst *EvmConfig) XXX_Merge(src proto.Message) {
+	xxx_messageInfo_EvmConfig.Merge(dst, src)
+}
+func (m *EvmConfig) XXX_Size() int {
+	return xxx_messageInfo_EvmConfig.Size(m)
+}
+func (m *EvmConfig) XXX_DiscardUnknown() {
+	xxx_messageInfo_EvmConfig.DiscardUnknown(m)
+}
+
+var xxx_messageInfo_EvmConfig proto.InternalMessageInfo
+
+func (m *EvmConfig) GetGasLimit() uint64 {
+	if m != nil {
+		return m.GasLimit
+	}
+	return 0
+}
+
 type Config struct {
 	AppStore             *AppStoreConfig `protobuf:"bytes,2,opt,name=app_store,json=appStore" json:"app_store,omitempty"`
+	Evm                  *EvmConfig      `protobuf:"bytes,3,opt,name=evm" json:"evm,omitempty"`
```

Comment:
How come Evm is after AppStore here, but the other way around in the proto?

Reply:
yeah forget to update protobuf, fixed